spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -37,21 +37,21 @@
       "defaultValue": true
     },
     {
-      "name": "spring.cloud.gcp.pubsub.enabled",
+      "name": "spring.cloud.gcp.metrics.enabled",
       "type": "java.lang.Boolean",
-      "description": "Auto-configure Google Cloud Pub/Sub components.",
+      "description": "Auto-configure Google Cloud Monitoring for Micrometer.",
       "defaultValue": true
     },
     {
-      "name": "spring.cloud.gcp.pubsub.reactive.enabled",
+      "name": "spring.cloud.gcp.pubsub.enabled",
       "type": "java.lang.Boolean",
-      "description": "Auto-configure Google Cloud Pub/Sub Reactive components.",
+      "description": "Auto-configure Google Cloud Pub/Sub components.",
       "defaultValue": true
     },
     {
-      "name": "spring.cloud.gcp.secretmanager.enabled",
```

Comment:
Looks like you lost `spring.cloud.gcp.secretmanager.enabled` in the merge.

Reply:
Ah no, it's still in here, the diff makes it look weird because things got reordered - but see line 76 of this file where `secretmanager` is.